### DISCUSSION TITLE
The "X" icon has not changed from the "Twitter" icon in the GitHub mobile app (iOS)

### DISCUSSION THREAD

**Discussion by darkotodoric** [community/community#103208] [Mobile] [answered]
### Select Topic Area

Bug

### Feature Area

Issues

### Body

The GitHub mobile app on iOS displays a "Twitter" icon instead of the 'X' icon. I am not sure about Android.

GitHub mobile app:
https://github.com/community/community/assets/7748474/5340ede0-3372-498b-8682-82cdb594bf34

Website:
https://github.com/community/community/assets/7748474/6e43efbe-c97d-4721-952e-bd8f8b89ccdb

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @darkotodoric ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by jsoref** [comment, marked as answer]
Duplicate of (answered):
* https://github.com/orgs/community/discussions/104722

**Reply by darkotodoric** [reply]
actually, this issue is older :)

**Reply by jsoref** [reply]
I get that. But a misfiled issue doesn't earn brownie points. The goal of issues should be to get things fixed. That issues has already achieved that goal for this issue, and thus you should mark this as a closed item so that we can free their 👀 for other issues.

**Reply by darkotodoric** [reply]
Waiting for the admin to do that, but ok. Done.
